plugins-compat-tester/src/test/java/org/jenkins/tools/test/PluginCompatTesterTest.java

```diff
@@ -74,4 +78,57 @@ public void testWithUrl() throws Throwable {
         PluginCompatTester tester = new PluginCompatTester(config);
 		tester.testPlugins();
 	}
+
+	@Test
+	public void testMatcher() throws Throwable {
+
+		String fileName = "WEB-INF/lib/jenkins-core-2.7.3-alpha-33.jar";
+		Matcher m = Pattern.compile(PluginCompatTester.JENKINS_CORE_FILE_REGEX).matcher(fileName);
+		assertTrue("No matches",m.matches());
```

Comment:
NIT: should check the capture group is also what you expect. and probably have some invalid cases.